iidm/iidm-xml-converter/src/test/resources/V1_0/staticVarCompensatorRoundTripRef.xml

```diff
@@ -5,7 +5,7 @@
             <iidm:busBreakerTopology>
                 <iidm:bus id="B1"/>
             </iidm:busBreakerTopology>
-            <iidm:generator id="G1" energySource="OTHER" minP="50.0" maxP="150.0" voltageRegulatorOn="true" targetP="100.0" targetV="400.0" bus="B1" connectableBus="B1">
+            <iidm:generator id="G1" energySource="OTHER" minP="50.0" maxP="150.0" voltageRegulatorOn="true" targetP="100.0" targetV="400.0" targetQ="0.0" bus="B1" connectableBus="B1">
```

Comment:
As a general good practice: files of previous XIIDM versions should not change except in extremely rare cases